docs/docs/representation_theory.md

```diff
@@ -0,0 +1,170 @@
+# Practical introduction to the representation theory of space groups
+
+The following note is a practical description of calculating character tables for symmorphic space groups and their use in constructing wave functions that are invariant under the same symmetries, with particular focus on some architectures used in NetKet. An accessible reference for more mathematical detail is [^bradley].
+
+[^bradley]: [The Mathematical Theory of Symmetry in Solids.](https://global.oup.com/academic/product/the-mathematical-theory-of-symmetry-in-solids-9780199582587?cc=gb&lang=en&) C. J. Bradley and A. P. Cracknell (1972). Clarendon Press, Oxford. (Chapter 3 is an accessible but comprehensive summary of the construction for both symmorphic and non-symmorphic groups; Chapter 4 covers every loose points of pure maths.)
+
+## Space groups and point groups
+
+A **space group** is the set of all symmetries of a system with discrete translational invariance. It consists of the same translations as well as symmetries that change its orientation (rotations, reflections, glides, screws, etc.). There is a finite number of possible space groups in any number of dimensions, which (and all the other objects we mention here) have been tabulated by crystallographers. In 2D, there are 17 space group (also known as *wallpaper groups*); in 3D, there are 230. We use $S$ to denote it.
+
+The **translation group** of the same system is the subgroup of the space group that consists of all translations by lattice vectors. It is a normal subgroup: it is intuitively clear that if $t$ is a lattice translation, and $g$ is any symmetry in the space group, then $g^{-1}tg$ is another pure lattice translation, since applying $g$ undoes any change in orientation that $g$ might have caused.
```

Comment:
```suggestion
The **translation group** of the same system is the subgroup of the space group that consists of all translations by lattice vectors. It is a normal subgroup: it is intuitively clear that if $t$ is a lattice translation, and $g$ is any symmetry in the space group, then $g^{-1}tg$ is another pure lattice translation, since applying $g^{-1}$ undoes any change in orientation that $g$ might have caused.
```